sandbox/handlers.js

```diff
@@ -133,6 +135,12 @@ async function batch_send(req, res, next) {
         return;
     }
 
+    if (req.body.data.attributes.routingPlanId === invalidRoutingPlanId) {
```

Comment:
This needs to be `req.body.sendingGroupId` instead of `req.body.data.attributes.routingPlanId` - the mock application receives the transformed POST body.